Projects/SelectTier/Sources/Navigation/SelectTierNavigation.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import SwiftUI
+import hCoreUI
+
+public class SelectTierNavigationViewModel: ObservableObject {
+    @Published public var isEditTierPresented = false
+    @Published public var isEditDeductiblePresented = false
+
+    var vm = SelectTierViewModel()
+
+    init() {}
```

Comment:
I think we should init SelectTierNavigation with the proper **contractId** and **source** (like its going the be expected from the 
```
enum TierSourceInputType {
    CHANGE,
    TERMINATION
}
```

After you do so, you can use it to set proper values when initing SelectTierViewModel inside this class